pkg/platform/packet/packet.go

```diff
@@ -223,7 +222,7 @@ func createTerraformConfigFile(cfg *config, terraformPath string) error {
 	t, err := t.Parse(terraformConfigTmpl)
 	if err != nil {
 		// TODO: Use template.Must().
-		return errors.Wrap(err, "failed to parse template")
+		return fmt.Errorf("parsing cluster template: %w", err)
```

Comment:
Should it be `cluster` or `Terraform config`

Reply:
I think I'll just change it to `parsing template`, as there is only one template parsed in this function.